### PR TITLE
data-json: fixing issue with null value

### PR DESCRIPTION
**Pull request by Garneauma**
Fixing issue where Data-JSON would throw an error when the value of a property is equal to `null`.

Related to WET-408

### CODE REVIEW

**Review by GormFrank — DISMISSED** [review]

**Review by duboisp — CHANGES_REQUESTED** [review]
Approved but I am going to add a test cases.

**Review by duboisp — APPROVED** [review]
Tested the fix and now there is a test example.

Before to merge, I will just wait for either @Garneauma or @GormFrank confirmation about the test cases that was added.

This change is a Patch

**Comment by duboisp** [comment]
Confirmed with @Garneauma the test-case addition I made is ok.

**Pull request merged**
